### DISCUSSION TITLE
Welcome to flutter_convenient_test Discussions!

### DISCUSSION THREAD

**Discussion by fzyzcjy** [fzyzcjy/flutter_convenient_test#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Raghavaraju-K** [comment]
Hello @fzyzcjy 

Hope you are doing good!

Could you please let me know the detailed architecture and design you have followed for this framework.?

Any help would be highly appreciated!

Thanks
Raghava

**Reply by fzyzcjy** [reply]
Sure! Feel free to create an issue and I will add it to my todolist and reply as soon as possible (though may need a bit of time since really busy now)
